### PR TITLE
Fat130 hei is no satisfaction score messaging

### PR DESCRIPTION
**Pull request by mapaux**

### CODE REVIEW

File: src/Web/Sfa.Das.Sas.ApplicationServices/Models/FrameworkProviderSearchResultsItem.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@ namespace Sfa.Das.Sas.ApplicationServices.Models
     public sealed class FrameworkProviderSearchResultsItem : IApprenticeshipProviderSearchResultsItem
     {
         public int Ukprn { get; set; }
+        public bool Hei { get; set; }
```
Comment (froberg) [inline src/Web/Sfa.Das.Sas.ApplicationServices/Models/FrameworkProviderSearchResultsItem.cs]: Change to IsHigherEducationInstitute or something better. 
:)
/c

File: src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs [hunk 2]
```diff
@@ -21,6 +21,21 @@ internal static string GetPercentageText(double? level)
             return $"{level}%";
         }
 
+        internal static string GetPercentageText(double? level, bool hei)
+        {
+            if (level == null && hei)
+            {
+                return "Not currently collected for this training organisation";
```
Comment (scottcowan) [inline src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs]: needs a unit test

File: src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs [hunk 3]
```diff
@@ -21,6 +21,21 @@ internal static string GetPercentageText(double? level)
...
+            {
+                return "no data available";
+            }
+
+            return $"{level}%";
+        }
+
         internal static string GetLocationAddressLine(TrainingLocation providerLocation)
         {
             return GetCommaList(
```
Comment (scottcowan) [inline src/Web/Sfa.Das.Sas.Web/Services/MappingActions/Helpers/ProviderMappingHelper.cs]: GetDeliveryOptionText needs some tests too (which you wrote back in july)

**Comment by scottcowan** [comment]
you still need to write tests for GetDeliveryOptionText

**Pull request merged**